lib/Experian/IDAuth.pm

```diff
@@ -309,7 +309,15 @@ sub _get_result_proveid {
 
     return unless $credit_reference and $kyc_summary;
 
-    my $decision = {matches => []};
+    my $decision = {matches => [], kyc_summary_score => 0};
+
+    # calculate kyc summary score
+    
+    if ( ( my $nameCount = $kyc_summary->findvalue('FullNameAndAddress/Count') ) > 0
+    && ( my $dobCount = $kyc_summary->findvalue('DateOfBirth/Count') ) > 0 )
+    {
+        $decision->{kyc_summary_score} = $nameCount + $dobCount;
```

Comment:
I think this newly added field is just a new name for `age_verified` with a new way of calculation. I think `age_verified`  is more descriptive.

Reply:
I would argue that `kyc_summary_score` is a more descriptive name for what the variable is while `age_verified` is more about what we use the variable for. If for example, we change the requirements for age verification, the `kyc_summary_score` name and variable still holds true and we'd need to update how we translate that field to age verification.